dali/test/python/test_pipeline.py

```diff
@@ -1654,3 +1654,102 @@ def test_invoke_serialize_error_handling_string():
 @raises(TypeError, "*define_graph*callable*")
 def test_invoke_serialize_error_handling_not_string():
     _identity_pipe().serialize(42)
+
+
+def check_dtype_ndim(dali_pipeline, output_dtype, output_ndim):
+    def ndim_dtype_matches(test_value, ref_value):
+        ref_value = ref_value if type(ref_value) is list else [ref_value]
```

Comment:
If `type(ref_value) is list` is False you zip `[ref_value]` of len 1 with test_value effectively ignoring all the remaining outputs.

Reply:
Fixed (I believe)